### DISCUSSION TITLE
Значение селекта в iframe окне

### DISCUSSION THREAD

**Discussion by f0gi** [totumonline/totum-ru-issues-and-discussions#562] [ОШБК!]
Добрый день! Подскажите пожалуйста, когда я нахожусь в простой таблице и по "кнопке" открываю другую таблицу в iframe окне > у меня в поле типа селект с возможностью добавления новых значений в поле.
Но когда я добавляю новое значение для селекта, его нельзя выбрать. Необходимо отменить создание строки, и добавить новую > тогда новое значение для селекта появится.
Возможно ли это как то решить?

**Comment by ttm-alexeypolunin** [comment]
Это ошибка где-то во фронте. Я записал, сроки исправления неизвестны и исправление будет в V5.

**Reply by f0gi** [reply]
Спасибо!
